src/platform/packages/shared/kbn-unified-data-table/src/components/data_table.tsx

```diff
@@ -457,6 +466,10 @@ interface InternalUnifiedDataTableProps {
    * @param row
    */
   getRowIndicator?: ColorIndicatorControlColumnParams['getRowIndicator'];
+  /**
+   * Custom bulk action
+   */
+  customBulkActions: CustomBulkActions;
```

Comment:
```suggestion
  customBulkActions?: CustomBulkActions;
```

Reply:
fixed in https://github.com/elastic/kibana/pull/230738/commits/bd8d541efe395f5cc2ce52d6619b345635b74f29